tests/topotests/static_routing_with_ebgp/test_static_routes_topo1_ebgp.py

```diff
@@ -0,0 +1,1343 @@
+
+        result = verify_rib(
+            tgen, addr_type, dut, input_dict_4, next_hop=nh, protocol=protocol
+        )
+        assert result is True, "Testcase {} : Failed \nError: Routes is"
+        " missing in RIB".format(tc_name)
+
+        step("Shut nexthop interface N2")
+        intf = topo["routers"]["r2"]["links"]["r1-link1"]["interface"]
+        dut = "r2"
+        shutdown_bringup_interface(tgen, dut, intf, False)
+
+        step(
+            " after shut of nexthop N1 , route become active with "
+            "nexthop N2 and vice versa."
+        )
+        nh = NEXT_HOP_IP["nh2"][addr_type]
+
+        result = verify_rib(
+            tgen,
+            addr_type,
+            dut,
+            input_dict_4,
+            next_hop=nh,
+            protocol=protocol,
+            expected=False,
+        )
+        assert result is not True, "Testcase {} : Failed \nError: Routes is"
+        " still present in RIB".format(tc_name)
+
+        nh = [NEXT_HOP_IP["nh1"][addr_type]]
+        dut = "r2"
+        protocol = "static"
+        result = verify_rib(
+            tgen, addr_type, dut, input_dict_4, next_hop=nh, protocol=protocol
+        )
+        assert result is True, "Testcase {} : Failed \nError: Routes is"
+        " missing in RIB".format(tc_name)
+
+        dut = "r3"
+        result = verify_bgp_rib(tgen, addr_type, dut, input_dict_4)
+        assert result is True, "Testcase {} : Failed \nError: Route is"
+        " missing in RIB".format(tc_name)
+
+        result = verify_rib(
+            tgen, addr_type, dut, input_dict_4, protocol=protocol, expected=False
+        )
+        assert result is not True, "Testcase {} : Failed \nError: Route is"
+        " still present in RIB".format(tc_name)
+
+        step("No shut nexthop interface N2")
+        dut = "r2"
+        shutdown_bringup_interface(tgen, dut, intf, True)
+
+        step(
+            "after shut of nexthop N1 , route become active "
+            "with nexthop N2 and vice versa."
+        )
+        nh = [NEXT_HOP_IP["nh1"][addr_type], NEXT_HOP_IP["nh2"][addr_type]]
+
+        result = verify_rib(
+            tgen, addr_type, dut, input_dict_4, next_hop=nh, protocol=protocol
+        )
+        assert result is True, "Testcase {} : Failed \nError: Routes is"
+        " missing in RIB".format(tc_name)
+
+        dut = "r3"
+        result = verify_bgp_rib(tgen, addr_type, dut, input_dict_4)
+        assert result is True, "Testcase {} : Failed \nError: Route is"
+        " missing in RIB".format(tc_name)
+
+        result = verify_rib(
+            tgen, addr_type, dut, input_dict_4, protocol=protocol, expected=False
+        )
+        assert result is not True, "Testcase {} : Failed \nError: Route is"
+        " still present in RIB".format(tc_name)
+
+        step("Reload the FRR router")
+        # stop/start -> restart FRR router and verify
+        stop_router(tgen, "r2")
+        sleep(5)
```

Comment:
What's the point of the sleep?

Reply:
Changes made are:
1. Removed the static sleep.
2. Base config is modified to use bgp aggressive timers.